.travis.yml

```diff
@@ -4,8 +4,8 @@ services:
   - docker
 
 before_install:
-  - docker build -t yast-alternatives-image .
+  - docker build -t yast-update-alternatives-image .
```

Comment:
as this breaks convention introduced in other similar PRs (<pkg_name>-image) and fixes already accepted update for docker I'd like to know what was wrong with the original image name?